### DISCUSSION TITLE
Date insertion

### DISCUSSION THREAD

**Discussion by danilshvalov** [nvim-orgmode/orgmode#539] [Q&A] [answered]
Is it possible to insert a date like in emacs?

<img width="596" alt="example" src="https://user-images.githubusercontent.com/57654917/228471489-6c708786-bdeb-496c-8044-8e21beae62ae.png">

**Comment by jgollenz** [comment]
Not that I'm aware of. Do you mind linking the orgmode docs for the feature? Thanks :slightly_smiling_face:

**Reply by danilshvalov** [reply]
I think it's described [here](https://orgmode.org/manual/The-date_002ftime-prompt.html).

**Comment by danilshvalov** [comment, marked as answer]
Implemented in #542
